src/Tribe/Aggregator/Processes/Batch_Imports.php

```diff
@@ -65,20 +70,14 @@ public function allow_batch_import( $service_supports_batch_push, $abstract ) {
 			return $service_supports_batch_push;
 		}
 
-		$parent_id = $abstract->post->post_parent;
-
-		// Make sure $parent_id is always an integer.
-		if ( $parent_id instanceof WP_Post ) {
-			$parent_id = $parent_id->ID;
-		}
+		$parent_id = $abstract->post->post_parent instanceof WP_Post
+			? $abstract->post->post_parent->ID
+			: $abstract->post->post_parent;
 
-		$allow_batch_pushing = get_post_meta(
-			$parent_id,
-			Record_Abstract::$meta_key_prefix . 'allow_batch_push',
-			true
-		);
+		$parent_record = Records::instance()->get_by_post_id( $parent_id);
```

Comment:
* :no_entry_sign: Expected 1 spaces before closing parenthesis; 0 found
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: cc8c7506d6b830edb18bfcf27fa21e45</sub>